src/network/core/tcp_http.cpp

```diff
@@ -68,17 +68,16 @@ NetworkHTTPSocketHandler::NetworkHTTPSocketHandler(SOCKET s,
 /** Free whatever needs to be freed. */
 NetworkHTTPSocketHandler::~NetworkHTTPSocketHandler()
 {
-	this->CloseConnection();
+	/* Virtual functions get called statically in destructors, so make it explicit to remove any confusion. */
+	this->NetworkHTTPSocketHandler::CloseSocket();
```

Comment:
Since `CloseSocket` is not virtual anymore, the explicit class is not needed anymore. Likewise for the comment.